lib/openstates/models/bill.rb

```diff
@@ -32,7 +32,7 @@ def api_method
       end
 
       def id_key
-        :bill_id
+        :id
```

Comment:
In matching the style of the other models, let's leave this as `bill_id`. Unless there is something I'm missing and this is necessary.